core-processor/src/common.rs

```diff
@@ -138,6 +140,7 @@ impl DispatchResult {
             system_reservation_context,
             page_update: Default::default(),
             allocations: Default::default(),
+            reply_sent: false,
```

Comment:
We need it in one place after program execution is over. It has to be saved since this is the only struct which is passed onto the processing.rs

Reply:
The value is also always reset to false, after every execution. Just the way we talked about. I cant just not initialize it or remove it from the struct or that would break the processing.rs

Reply:
It's not always reset to false. But this fn is used only for case of non-executing with success. Specify it in a comment or add `reply_sent: bool` argument to the fn

Reply:
This function is only used to create a dispatchresult when there is no code execution whatsoever, ie, the export we wanted to call was not found (so we send a message but handle method doesnt exist), therefore a reply can never be sent. I can add a comment if you feel that is necessary, but due to this usecase being the way it is, i dont think it is necessary.

Reply:
Nobody will remember this small thing if any new use-case of the fn will appear some time soon, so small comment is a must-have